test/integration/node-specific/client_close.test.ts

```diff
@@ -128,25 +329,138 @@ describe.skip('MongoClient.close() Integration', () => {
       describe('ConnectionPool', () => {
         describe('Node.js resource: minPoolSize timer', () => {
           describe('after new connection pool is created', () => {
-            it.skip('the minPoolSize timer is cleaned up by client.close()', async () => {});
+            it('the minPoolSize timer is cleaned up by client.close()', async function () {
+              const run = async function ({ MongoClient, uri, expect, getTimerCount }) {
+                const client = new MongoClient(uri, { minPoolSize: 1 });
+                let minPoolSizeTimerCreated = false;
+                client.on('connectionPoolReady', () => (minPoolSizeTimerCreated = true));
+                await client.connect();
+
+                expect(minPoolSizeTimerCreated).to.be.true;
+
+                const servers = client.topology?.s.servers;
+
+                function getMinPoolSizeTimer(servers) {
+                  for (const server of servers) {
+                    return server[1].pool.minPoolSizeTimer;
+                  }
+                }
+                // note: minPoolSizeCheckFrequencyMS = 100 ms by client, so this test has a chance of being flaky
+                expect(getMinPoolSizeTimer(servers)).to.exist;
+
+                await client.close();
+                expect(getMinPoolSizeTimer(servers)).to.not.exist;
+                expect(getTimerCount()).to.equal(0);
+              };
+              await runScriptAndGetProcessInfo('timer-min-pool-size', config, run);
+            });
           });
         });
 
         describe('Node.js resource: checkOut Timer', () => {
-          // waitQueueTimeoutMS
           describe('after new connection pool is created', () => {
-            it.skip('the wait queue timer is cleaned up by client.close()', async () => {});
+            it('the wait queue timer is cleaned up by client.close()', async function () {
+              const run = async function ({
+                MongoClient,
+                uri,
+                expect,
+                sinon,
+                sleep,
+                getTimerCount,
+                timers
+              }) {
+                const waitQueueTimeoutMS = 1515;
+
+                // configure failPoint
+                const utilClient = new MongoClient(uri);
```

Comment:
Can you use a beforeEach/afterEach to make sure we clean up the fail point? A great way to make sure only the client in the test responds is to set an appName

Reply:
Yep, made the change!